requirements-test.txt

```diff
@@ -1,2 +1,4 @@
 pytest
-pytest-cov
\ No newline at end of file
+pytest-cov
+ml_wrappers
```

Comment:
these should be in package dependencies and not test

Reply:
added to requirements.txt in the outer level of this repo and removed from test.